Doc/whatsnew/3.14.rst

```diff
@@ -2689,99 +2698,102 @@ Deprecated
     * :func:`asyncio.set_event_loop_policy`
 
     Users should use :func:`asyncio.run` or :class:`asyncio.Runner` with
-    *loop_factory* to use the desired event loop implementation.
+    the *loop_factory* argument to use the desired event loop implementation.
 
-    For example, to use :class:`asyncio.SelectorEventLoop` on Windows::
+    For example, to use :class:`asyncio.SelectorEventLoop` on Windows:
 
-      import asyncio
+    .. code:: python
```

Comment:
Any particular reason for changing to the longer, explicit `:\n\n.. code-block: python` over `::`?

RST is pretty tricky so in general I think we should favour the easier to use ones.

Reply:
reST is equally as tricky to use as MD! (how many spaces after the end of a line to force a line break, and in which flavours?)

I prefer using the directive form for explicitness. For example, if illustrating a traceback or console session, one wants to use `.. code-block:: pytb/pycon`, but this is impossible with the `::` syntax.

On the user-friendliness argument, I'd also posit that the explicit directive form is easier to comprehend for a newcomer. `::` implicitly treats the following indented block as Python code and only displays the first colon, but this is perhaps surprising behaviour until internalised!

A

Reply:
Would it be possible to use `code-block` then? I don't think we use `code` a lot (~60 vs >500)

> but this is impossible with the :: syntax.

Strictly speaking, no if you set `.. highlight:: pycon` before using `::` and add `.. highlight:: default` afterwards.

Reply:
I've changed to code-block